zeton/models.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3 as sql
+
+
+def wprowadzUzytkownika(login_uzytkownika, haslo_uzytkownika):
+    con = sql.connect("db.sqlite")
+    cur = con.cursor()
+    cur.execute("INSERT INTO uzytkownicy (login, haslo) VALUES (?,?)", (login_uzytkownika, haslo_uzytkownika))
+    con.commit()
+    con.close()
+
+
+def wyciagnijUzytkownika():
+    con = sql.connect("db.sqlite")
+    cur = con.cursor()
+    cur.execute("SELECT login, haslo FROM uzytkownicy")
+    uzytkownicy = cur.fetchall()
+    con.close()
+    return uzytkownicy
+
+
+def create_tables():
+    query_punkty_uczniow = """create table punkty_uczniow (
+        id_ucznia integer primary key autoincrement,
+        suma_punktow integer not null,
+        szkolny_rekord_tygodnia integer not null
+        );"""
+
+    query_uzytkownicy = """create table uzytkownicy (
+      id_uzytkownika integer primary key autoincrement,
+      login text not null,
+      haslo text not null
+      );"""
+
+    con = sql.connect("../db.sqlite")
```

Comment:
Plik z bazą trzymamy w roocie, czy w katalogu ./zeton ?
Na razie używałem tego, który znalazłem.

Reply:
właśnie nie wiem, myślałam o katalogu ./zeton. 
Generalnie przydadzą nam się wskazówki do struktury :) (np jak teraz najlepiej porozdzielac ten nasz glowny plik zeton_glowny, zeby sie nie pogubic; gdzie trzymac zapytania bazodanowe itp)

Reply:
Na razie zostawiam tak jak jest. Jak Leszek się zgodzi to można mergować.